src/Mvc/Mvc.DataAnnotations/src/MvcDataAnnotationsLocalizationOptions.cs

```diff
@@ -19,7 +19,7 @@ public class MvcDataAnnotationsLocalizationOptions : IEnumerable<ICompatibilityS
         /// <summary>
         /// The delegate to invoke for creating <see cref="IStringLocalizer"/>.
         /// </summary>
-        public Func<Type, IStringLocalizerFactory, IStringLocalizer> DataAnnotationLocalizerProvider;
+        public Func<Type, IStringLocalizerFactory, IStringLocalizer> DataAnnotationLocalizerProvider = null!;
```

Comment:
Huh, weird that this is a field

Reply:
Oh wow, did not notice that. API-review fail :)